docs/reference-guide/enterprise-features.md

```diff
@@ -0,0 +1,43 @@
+---
+id: enterprise-features
+title: Enterprise features
+sidebar_label: Enterprise features
+---
+
```

Comment:
can you add some kind of intro here so we don't start with a title right at the beginning?